README.md

```diff
@@ -35,10 +35,14 @@ stage('Publish') {
     APPCENTER_API_TOKEN = credentials('appcenter-api-token')
   }
   steps {
-    appCenter apiToken: APPCENTER_API_TOKEN, ownerName: 'owner-name', appName: 'app-name', pathToApp: 'path/to/app.apk'
+    appCenter apiToken: APPCENTER_API_TOKEN,
+            ownerName: 'owner-name',
+            appName: 'app-name',
+            pathToApp: 'path/to/app.apk',
+            distributionGroup: 'Collaborators'
```

Comment:
In an upcoming PR I'm looking to support a comma separated list of distribution groups so could you change this variable to `distributionGroups`. Or alternatively I can include these changes in the PR that I am working on?

Reply:
That would be a nice change as well 👍 For my own project though, I'd have everything I need when I can release to a single distribution group.

Reply:
Yeah that's fair enough. Thanks for the contribution. I am happy to merge this in and you can get the built artefact from the Jenkins CI server if you need to use it straight away. Just note that I was going to wait until I the changes I spoke about above were merged (hopefully soon!) before attempting a 0.2 release. So in between this snapshot build and 0.2 your API for the DSL will change.